tests/test_blocks_toolbox.js

```diff
@@ -0,0 +1,955 @@
+/**
+ * @license
+ * Copyright 2014 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Test blocks toolbox has a category menu and an auto-closing flyout.
+ * The blocks in this toolbox reflect block configurations not used by
+ * the standard predefined blocks, and so test alternative block rendering
+ * code paths.
+ */
+
+
+var alignCategory = {
```

Comment:
Nice, I like this breakup into categories. Leaves us the opportunity to add a comment of what each category includes.